lib/default-html.js

```diff
@@ -1,13 +1,14 @@
 'use strict';
 
-module.exports = ({ renderCssLinks, renderJsBundles }) => {
+module.exports = ({ renderCssLinks, renderJsBundles, polyfillScript }) => {
   return `
     <!DOCTYPE html>
     <html lang="en">
     <head>
       <meta charset="UTF-8">
       <meta name="viewport" content="width=device-width, initial-scale=1.0">
       <title>Underreact app</title>
+      ${polyfillScript()}
```

Comment:
Hm, I'm a little worried about the need to manually include this. The problem that will arise is that, unlike the other `render*` functions that are being passed, the user developing on Chrome locally won't know that they missed a step. If they don't QA properly, they won't end up knowing until they realize at some point that the page is blank on IE11.

Reply:
I think you have a valid fear. I will go ahead and make this an automatic insertion in a new PR.